.github/workflows/ci.yml

```
@@ -1,36 +1,27 @@
 name: CI
 
-on:
-  push:
-    branches:
-      - master
-      - main
-      - "v*"
-    tags:
-      - "v*"
-  pull_request:
-  schedule:
-    - cron: '0 3 * * *' # daily, at 3am
+on: [push]
```

We might also want a `pull_request` event here.

```suggestion
on:
  push:
  pull_request:
```

When you have an already existing branch and then open a PR, the workflows might not be kicked off without a push event afterwards.